index.js

```diff
@@ -5,6 +5,8 @@ const YAML = require('yaml');
 const myToken = process.env.TOKEN;
 const octokit = new github.GitHub(myToken);
 
+const awsEcrUrl = 'public.ecr.aws/v6b8u5o6';
```

Comment:
I think I would name the variable `AWS_ECR_URL` to show that this a real const/config value